dev.sh

```diff
@@ -11,7 +11,7 @@ if [ ! -f "$TOOL_DEST/task" ]; then # check for local installation
     fi
 fi
 
-export PATH="$PATH:$TOOL_DEST:$TOOL_DEST/kubebuilder/bin"
+export PATH="$TOOL_DEST:$TOOL_DEST/kubebuilder/bin:$PATH"
```

Comment:
We prepend here because we want to override whatever tools they have in their environment path already to ensure the right ones are used.